AndroidXCI/lib/src/main/kotlin/dev/androidx/ci/testRunner/dto/TestRun.kt

```diff
@@ -84,7 +84,7 @@ internal class TestRun(
                         listOf(it.key, it.value)
                     },
                     "additionalApks" to deviceSetup?.additionalApks?.map {
-                        it.apkInfo.idHash
+                        it.gcsPath.path
```

Comment:
i was thinking about the sorting here.

you've mentioned that the order we send them to FTL might be important.
if it is important, then we should maybe preserve the order here as well? (so that if they are re-ordered, we re-run the test?)

Reply:
if thats the case, when you remove `sorted`, please add a comment for the next person that this is intentionally not ordered because the order matters.

Reply:
done